### PR TITLE
golangci-lint pass number 2

### PR DESCRIPTION
**Pull request by baude**
clean up and prepare to migrate to the golangci-linter

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/netns/netns_linux.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ package netns
 import (
 	"crypto/rand"
 	"fmt"
+	"github.com/sirupsen/logrus"
```
Comment (mheon) [inline pkg/netns/netns_linux.go]: Import ordering

**Comment by rhatdan** [comment]
After fix the one NIT, LGTM

**Pull request merged**